cmake/FindHIP.cmake

```diff
@@ -555,7 +560,13 @@ macro(HIP_ADD_EXECUTABLE hip_target)
         list(REMOVE_ITEM _sources ${_source_files})
     endif()
     if("x${HCC_HOME}" STREQUAL "x")
-        set(HCC_HOME "/opt/rocm/hcc")
+        if (DEFINED $ENV{ROCM_PATH})
+            set(HCC_HOME "$ENV{ROCM_PATH}/hcc")
+        elseif( DEFINED $ENV{HIP_PATH})
+            set(HCC_HOME "$ENV{HIP_PATH}/../hcc")
```

Comment:
You need to test if HIP_PATH is part of ROCm installation. So you need to test if $HIP_PATH/../.info/version exists before using ENV(HIP_PATH)/../hcc.

Reply:
done with checking hipconfig